source/SIL.AppBuilder.Portal.Frontend/src/ui/routes/project/reviewers/item.tsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react';
+import { compose } from 'recompose';
+import { translate, InjectedTranslateProps as i18nProps } from 'react-i18next';
+import { ResourceObject } from 'jsonapi-typescript';
+import { Icon } from 'semantic-ui-react';
+
+import { ReviewerAttributes } from '@data/models/reviewer';
+import { attributesFor } from '@data';
+import { REVIEWERS_TYPE } from '@data';
+import { withDataActions, IProvidedProps } from '@data/containers/resources/reviewer/with-data-actions';
+
+interface Params {
+  reviewer: ResourceObject<REVIEWERS_TYPE, ReviewerAttributes>;
+}
+
+type IProps =
+  & Params
+  & i18nProps
+  & IProvidedProps;
+
+class ReviewerItem extends React.Component<IProps> {
+
+  removeReviewer = (reviewer) => () => {
+
+    const { removeRecord } = this.props;
+
+    try {
+      removeRecord(reviewer);
```

Comment:
await?

Reply:
We don't need it because we are using await inside removeRecord